src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -324,6 +324,22 @@ public static class ClientOperations
                         ValidateContent(content, await m.Content.ReadAsStringAsync());
                     }
                 }),
+                
+                ("POST Duplex Dispose",
+                async ctx =>
+                {
+                    // try to reproduce conditions described in https://github.com/dotnet/corefx/issues/39819
+                    string content = ctx.GetRandomString(0, ctx.MaxContentLength);
+                    Version httpVersion = ctx.GetRandomHttpVersion();
+
+                    using (var req = new HttpRequestMessage(HttpMethod.Post, "/duplex") { Version = httpVersion, Content = new StringDuplexContent(content) })
+                    using (HttpResponseMessage m = await ctx.SendAsync(req, HttpCompletionOption.ResponseHeadersRead))
+                    {
+                        ValidateHttpVersion(m, httpVersion);
+                        ValidateStatusCode(m);
+                        // Leave scope (and dispose of the response), potentially before the client has had the chance to read the entire content
```

Comment:
Can we clarify the comment here?

The key behavior here is not whether "the client has had the chance to read the entire content". It's that the user has disposed the content before reading to EOF. So something like:

// Cause the response to be disposed without reading the response body, which will cause the client to cancel the request

Reply:
@geoffkizer done